Chapter-2-modules-separation/Src/Contracts/Fitnet.Contracts.Api/ContractsModule.cs

```diff
@@ -1,13 +1,17 @@
 namespace EvolutionaryArchitecture.Fitnet.Contracts.Api;
 
+using System.Reflection;
+using Common.Api.Validation.Requests;
 using EvolutionaryArchitecture.Fitnet.Common.Infrastructure.Modules;
 using Infrastructure;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Sign;
 
 public static class ContractsModule
 {
+    private static Assembly CurrentModule => typeof(SignContractRequest).Assembly;
```

Comment:
Add spacing and maybe it makes more sense to use a local variable for current assembly as you use it just in one place (line 28)

Reply:
Ok